libs/ui-watt/project.json

```diff
@@ -25,8 +25,9 @@
     "storybook": {
       "executor": "@nrwl/storybook:storybook",
       "options": {
-        "uiFramework": "@storybook/angular",
         "port": 4400,
+        "projectBuildConfig": "app-dh",
```

Comment:
suggestion: By looking at a spec file in Nx (https://github.com/nrwl/nx/blob/dcb839c49d102f5d50b5d54875750120c2090faf/packages/storybook/src/migrations/update-13-4-6/__snapshots__/set-project-build-config.spec.ts.snap#L109), it seems like `projectBuildConfig` should be set to something like `ui-watt:build-storybook`.